### ISSUE TITLE
Feature request: being able to use an url instead of a filepath

### ISSUE THREAD

**Issue by christian-nielsen** [kirschbaum-development/laravel-openapi-validator#20]
I am using: https://github.com/dedoc/scramble and it generates the yaml specification in realtime.

Would we be nice to use this package with an url/route

**Comment by luisdalmolin** [comment]
Yes, that would be great indeed. We can maybe take a look at this at some point, but are are also definitely open to PRs.
